docs/sources/alerting/silences/edit-silence.md

```diff
@@ -0,0 +1,14 @@
++++
+title = "Edit silences"
+description = "Edit alerting silences"
+keywords = ["grafana", "alerting", "silence", "mute"]
+weight = 450
+aliases = ["/docs/grafana/latest/alerting/unified-alerting/silences/"]
++++
+
+# Edit silences
+
```

Comment:
no intro or before you begin?

Reply:
I am not sure what information will be useful here. Suggestions?